### PR TITLE
NEWS.md: update for 0.8.1

### PR DESCRIPTION
**Pull request by johanmalm**
For Friday

### CODE REVIEW

**Comment by Consolatis** [comment]
LGTM

**Comment by johanmalm** [comment]
Thanks. Started doing links for the pulls/issue. Sometimes GH recognizes it automatically, but doesn't seem to work in `NEWS.md`. If people like it I can carry on doing it.

**Comment by Consolatis** [comment]
Maybe we could throw some script at it and convert all existing PR numbers automatically so you don't have to do it manually.

**Comment by tokyo4j** [comment]
I think the line `- Fix button scaling issues (#2207, #2225)` should be separated into:

```
- Fix rounded hover effect on titlebar buttons when the window is tiled or maximized (#2207) 
- Fix button scaling issue (#2225)
```

Sorry for my unhelpful commit message.

Other than that, LGTM.

**Comment by johanmalm** [comment]
> Maybe we could throw some script at it and convert all existing PR numbers automatically so you don't have to do it manually.

Yep. Although some relate to issues. I'll just do the numbers for 0.8.1 and stop at that.

**Comment by Consolatis** [comment]
> Yep. Although some relate to issues

That shouldn't matter for GH as it redirects to discussions / issues appropriately. See for example `https://github.com/labwc/labwc/pull/2258` (discussion) or `https://github.com/labwc/labwc/pull/2227` (issue).

File: NEWS.md [hunk 1]
```diff
@@ -44,33 +45,34 @@ Notes to package maintainers:
...
   the `meson setup` command in the build script for the next release.
-- PRs #1716 and #2205 add labwc xdg-portal configuration, modify `labwc.desktop`
-  and amend `XDG_CURRENT_DESKTOP` which should enable better out-of-the-box
-  support for xdg-desktop-portal, but if you already ship a custom setup for
-  this you or have different requirements, please review this change.
+- PRs [#1716] and [#2205] add labwc xdg-portal configuration, modify
+  `labwc.desktop` and amend `XDG_CURRENT_DESKTOP` which should enable better
+  out-of-the-box support for xdg-desktop-portal, but if you already ship a
+  custom setup for this you or have different requirements, please review this
+  change.
```
[the remark this answers is not in the document]
Comment (johanmalm) [reply NEWS.md]: Done. Thanks.

**Pull request merged**